### ISSUE TITLE
Lumina Hours Tracking

### ISSUE THREAD

**Issue by benloh** [netcreateorg/netcreate-2018#186]
Total: 189  Used: 32.25

Oct 2021: 1
Nov 2021: 31.25

**Comment by benloh** [comment]
@jdanish @kalanicraig Just to keep everything in one place, I'll update this issue monthly with our burn rate so you have some sense of how much is left.

The total project is 189 hours.  As of Nov 30, we've spent 31.25 hours.  So I think we're in very good shape.  Things are progressing quite quickly and efficiently.

**Comment by kalanicraig** [comment]
Fantastic.

I tested export several times and the only thing I see happening is with the ID label in the nodes and edges export. The nodes “id” field needs to be uppercase “ID” and the ID field for the edges should be at the end of the export column list. The Source and Target ID fields in the edge table need to be labeled with the sentence case. 

With those changes, it should be exactly what we need.

Looking forward to import!

**Comment by benloh** [comment]
@jdanish @kalanicraig January burn rate update.

The total project is 189 hours. As of Jan 31, we've spent 104.5 hours and 84.5 hours remain.  We have plenty of headroom.

**Comment by jdanish** [comment]
Awesome. I am sure Kalani has a list, but can you remind me what is on the feature list to still do?  I believe import and optimization. Was there anything else? 

Joshua

----
Joshua A. Danish (he, him, his) (https://www.mypronouns.org/ <https://www.mypronouns.org/>)

***@***.***

http://www.joshuadanish.com

**Comment by benloh** [comment]
Yes Import and Optimization are the remaining large features.  There are a number of minor bug fixes.  Current working list is here: https://github.com/netcreateorg/netcreate-2018/projects/3

So far another 27 hours have been burned in February to date.  I expect import to eat up another 20 depending on how refined we go.  So we still have some room, depending on how far we go with optimization.

Now's a good time to brainstorm any other key changes you'd like to make.

**Comment by jdanish** [comment]
Awesome!  I’ve got many I’d love to see but will talk to Kalani / defer to Kalani as we should pick what will be most useful for Lumina.

Joshua


----
Joshua A. Danish (he, him, his) (https://www.mypronouns.org/ <https://www.mypronouns.org/>)

***@***.***

http://www.joshuadanish.com

**Comment by kalanicraig** [comment]
Oh! A force-unlock button on a locked node or edge usable only by an administrator. That would be really helpful. We still occasionally run into a locked node or edge (when someone clicks off it without hitting close or cancel) but no good repeatable process to force the behavior. An admin button would help get around that without a restart.

**Comment by benloh** [comment]
Noted, thanks!  I've created issues for them.

**Comment by benloh** [comment]
@jdanish Feel free to add the "many I'd like to see" changes to the Project board: https://github.com/netcreateorg/netcreate-2018/projects/3
Just pile them up at the bottom of the list.  Some might be easy 15 minuters.

**Comment by benloh** [comment]
@jdanish @kalanicraig February 2022 burn rate update.

The total project is 189 hours. As of Feb 28, we've spent 136.25 hours and 52.94 hours remain. Cleaning up import will likely eat up most of this.  Getting a refined usable import involves a LOT of moving pieces.

**Comment by jdanish** [comment]
Cool. Thanks for the update!

----
(from my iPhone)

Joshua Danish
http://www.joshuadanish.com

**Comment by kalanicraig** [comment]
Yes, thank you! Import working well goes a long way.

**Comment by jdanish** [comment]
Thanks! To be clear, does that mean the to-do list I just saw posted and potentially the two bugs I just reported? We still need to bang on import a bit this week to make sure it seems solid.

Thanks,

Joshua

----
Joshua A. Danish (he, him, his) (https://www.mypronouns.org/ <https://www.mypronouns.org/>)

***@***.***

http://www.joshuadanish.com

**Comment by benloh** [comment]
Yeah, I don't want to leave it in an unstable state, so we'll eat a few hours.

**Comment by jdanish** [comment]
Thanks - appreciate that. Mostly was checking so we know what to expect / not expect or if we need to prioritize. Thanks!
